components/util/opts.rs

```diff
@@ -811,6 +820,7 @@ pub fn from_cmdline_args(args: &[String]) -> ArgumentParsingResult {
         dump_display_list: debug_options.dump_display_list,
         dump_display_list_json: debug_options.dump_display_list_json,
         dump_layer_tree: debug_options.dump_layer_tree,
+        enable_recording: debug_options.enable_recording,
```

Comment:
There is a similar feature for dumping layout frame tree called `trace_layout`, maybe we should follow the convention and call this `trace_webrender` or `record_webrender`, or even `record_display_list`.